.github/workflows/update_submodules.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   update-submodules:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
Does this change fix [this](https://github.com/near/near-discovery/actions/workflows/update_submodules.yml) failing workflow?

Reply:
this along with the addition of the expected REPO_TOKEN should fix it.